tmpl/wiki/packages.md

```diff
@@ -24,137 +27,131 @@ We make heavy use of the following tools:
+MirageOS releases are published via github. First log into your account and create
+a GitHub API token if you haven't already. Store it in a file (e.g. `~/.config/dune/github.token`).
 If on a multi-user machine, ensure the privileges are set to prevent other users
 from reading it.
 
 Before releasing anything it's a good idea to review the outstanding issues.
 Perhaps some can be closed already? Maybe a `CHANGES.md` entry is missing?
 
 When ready to go, create a branch from `master` and edit the `CHANGES.md` file
-to list the interesting changes made since the last release. Make a PR for this
+to list the interesting changes made since the last release. Make a pull request (PR) for this
 update. The CI will run which is a useful final check that the code still builds
 and the tests still pass.
 (It's
 ok to skip this if the CI was working fine a few moments ago when you merged
-another PR).
+another PR). If you include `[ci skip]` in your commit message, the CI will not be run.
 
 When the `CHANGES.md` PR is merged, pull it into your local `master` branch.
 
-Read `topkg help release` to have an overview of the full release workflow.
-You need to install `odoc`, `topkg-jbuilder` and `opam-publish` to be installed.
-Run `opam-publish` once to generate a release token.
+Read `dune-release help release` to have an overview of the full release workflow.
+You need to have `odoc` installed to generate the documentation.
 
 Type:
 
 ```
-topkg tag
+dune-release tag
 ```
--- topkg will extract the latest version from the `CHANGES.md` file, perform
+-- dune-release will extract the latest version from the `CHANGES.md` file, perform
 version substitutions and create a local tag.
 
 Type:
 
 ```
-topkg distrib
+dune-release distrib
 ```
--- topkg will create a release tarball.
+-- dune-release will create a release tarball.
 
-Install `odoc,` topkg-jbuilder Type:
+Install `odoc` and type:
 
+```
+dune-release publish --dry-run
+```
+-- dune-release will build the documentation (fix all the warnings).
 
+Type:
 ```
-topkg publish
+dune-release publish
 ```
--- topkg will push the tag, create a release and upload the release tarball.
+-- dune-release will push the tag, create a release and upload the release tarball.
 It will also build the docs and push them online.
 
-If you have the [multi-package release rules](https://github.com/mirage/mirage-block/blob/master/Makefile#L12) in your Makefile,
-and assumning that you have a clone on `ocaml/opam-repository` in `../opam-repository`, you can then type:
-
+Type
 ```
-make opam-pkg
+dune-release opam pkg
+dune-release opam submit
 ```
 
 -- this will add new files in your opam-repository clone. `git commit` and push them to your fork on GitHub
 and open a new pull-request.
 
-If you only have one package in your repository, you can simply write:
+You can simply write:
 
 ```
-topkg tag && topkg bistro
+dune-release tag && dune-release bistro
```

Comment:
you don't need `bistro` here, `dune-release tag && dune-release` should just work.